### ISSUE TITLE
Unable to update entry due to incorrect validation

### ISSUE THREAD

**Issue by Stephen-Gates** [okfn/opendatacensus#773]
I'm trying to update http://australia.census.okfn.org/submit/?place=nsw&dataset=timetables
but get 
<img width="478" alt="screenshot 2016-08-02 20 49 12" src="https://cloud.githubusercontent.com/assets/9379524/17326167/d8cefb92-58f2-11e6-8059-9e5eafeb6acd.png">

I believe the changes I'm making to the values should not be errors.

**Comment by pwalsh** [comment]
@brew pls take a look

**Comment by brew** [comment]
It's not very clear in the current entry form, but some questions, if false, require other questions to also be answered false. e.g. If 'Publicly available' is answered 'No', then 'Free', 'Online', 'Open Licence' and 'Bulk' should also be 'No'. [1]

@Stephen-Gates Do you think this is what's happening with your submission?

[1] This is hard-coded in a validator: https://github.com/okfn/opendatacensus/blob/28fe2e5b949a23043e3ce5849abd96fdd27f130e/census/controllers/utils.js#L30

**Comment by Stephen-Gates** [comment]
@brew I think that's what's happening but I also think the validation is wrong. The dataset is not publicly available, you have to register. Once you've registered, it is free, online, available in bulk and openly licenced.

**Comment by KateRogers** [comment]
@Stephen-Gates agreed, this is something that we are changing this year. There will be a subquestion, if no, where users can specify the restriction to access (for example, registration) and then can continue on with the survey.
